gateway/main/settings.py

```diff
@@ -287,12 +287,24 @@
     ),
 }
 RAY_CLUSTER_WORKER_REPLICAS = int(os.environ.get("RAY_CLUSTER_WORKER_REPLICAS", "0"))
+RAY_CLUSTER_WORKER_REPLICAS_MAX = int(
```

Comment:
I remember we had a `readme` doc with all settings. I cannot find it for some reason 🤔 
If would be great to have all this settings specified in a document

Reply:
I'll look for it.